src/Core/Serialization/ProjectVersionMigrator.cs

```diff
@@ -0,0 +1,49 @@
+using System;
```

Comment:
Needs a banner, you can copy it from any other source file and put your name in it.

Reply:
Thats right, sorry! Didn't notice that because VS always collapse them for me.